### ISSUE TITLE
Quality loss comparing Dorado and Guppy with the same sample

### ISSUE THREAD

**Issue by fernanarr** [nanoporetech/dorado#188]
Hi,

I've made a test with a sample of 4000 sequences, running the basecalling using Dorado and Guppy.
Checking quality with NanoPlot, I've realized that Dorado has lost quality comparing to Guppy

Dorado:
![nanoplot_dorado](https://github.com/nanoporetech/dorado/assets/100858534/1349f877-0893-4a3d-8c8f-d6a0864a7f72)
Guppy:
![nanoplot_guppy](https://github.com/nanoporetech/dorado/assets/100858534/a708124d-1a83-46f4-85df-da0907db6c4c)

I've made the basecalling using dna_r10.4.1_e8.2_400bps_sup@v4.1.0 with a batchsize of 96 (my GPU wasn't able to run Dorado with full batchsize because only 7,6 GB were available and you helped me with that in another issue)

Is it normal to lose quality comparing to Guppy basecalling? is it caused by the conversion from fast5 to pod5? or perhaps is it caused by the batchsize I'm forced to use?

Thanks in avance for your help

Regards

**Comment by iiSeymour** [comment]
@fernanarr qscores are rough measure that correlate with alignment accuracy. Dorado and Guppy calculate these in slightly different ways and I don't think this is an issue here _(i.e. there is no loss of quality)_ . Are you able to align these calls and look at alignment accuracy?

**Comment by fernanarr** [comment]
Hi @iiSeymour ,

Actually we haven't got any reference to align with, because there is no clue about what might be in the reads obtained.

As Dorado uses pod5 files, which saves a lot of disk space, and the processing is faster than Guppy, we decided to run the same file with both softwares in order to compare the output. We are just interested in the quality report.

Due to the results, I've oppened the issue to check if it was a normal performance by Dorado or if any of my steps could have had caused the quality difference.

Thanks for your response and for your help as always.
